pkg/nodeconfig/nodeconfig.go

```diff
@@ -136,8 +136,13 @@ func NewNodeConfig(clientset *kubernetes.Clientset, clusterServiceCIDR, vxlanPor
 			"creating new node config")
 	}
 
+	clusterDNS, err := cluster.GetClusterDNS(clusterServiceCIDR)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error processing CIDR IP range: %s", clusterServiceCIDR)
```

Comment:
`error getting cluster DNS from service CIDR` is more accurate

Reply:
+1